examples/kitchen-sink.yaml

```diff
@@ -12,358 +12,404 @@ file_format: "0.1"
 # to ensure the SDK isn't disabled, the default value when this is not provided
 # is for the SDK to be enabled.
 #
-# Environment variable: OTEL_SDK_DISABLED
+# Environment variable: OTEL_CONF_DISABLED
 disabled: false
 
 # Configure general attribute limits. See also tracer_provider.limits, logger_provider.limits.
 attribute_limits:
   # Configure max attribute value size.
   #
-  # Environment variable: OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT
+  # Environment variable: OTEL_CONF_ATTRIBUTE_LIMITS__ATTRIBUTE_VALUE_LENGTH_LIMIT
```

Comment:
do you think the `_CONF_` portion is necessary?

Reply:
It seemed like a good way to clearly distinguish between this new environment variable schema and the old. Not strictly required, but the common prefix catches the eye and there is no mistaking which scheme it refers to.

We of course should weigh this against characters it adds to every single environment variable.

Reply:
I'm personally in favor of dropping it if the plan is for these to be the (one set of) recommended env vars going forward